src/ZProc.ml

```diff
@@ -201,3 +202,18 @@ let normalize (expr : string) : string =
   in match parse expr with
      | Done (sexp, _) -> Sexp.to_string_hum (helper (sexp))
      | _ -> ""
+
+let rec build_arguments (vals : value list) : string = 
+  match vals with 
+    | [] -> ""
+    | v :: r -> " " ^ (Types.serialize_value v) ^ (build_arguments r)
```

Comment:
We could remove this, and replace calls to this function everywhere with `List.to_string_map vals ~sep:" " ~f:Types.serialize_value`.
We would need `open Utils` in those files.